package.json

```diff
@@ -25,7 +25,7 @@
     "build": "yarn turbo run package",
     "install-frozen-lockfile": "./scripts/install-frozen-lockfile.sh",
     "dev-server": "yarn turbo run dev --parallel",
-    "postinstall": "husky install",
+    "postinstall": "husky install && git submodule init && git submodule update",
```

Comment:
maybe we should do yarn setup:venv in here too. WDYT?

Reply:
Good idea. Just one step to get up and running.

Reply:
I'll have more of a think about it and follow up.